release/nightly_tests/nightly_tests.yaml

```diff
@@ -315,3 +315,12 @@
     timeout: 7200
     prepare: python wait_cluster.py 32 1000
     script: python dask_on_ray/dask_on_ray_sort.py --nbytes 1_000_000_000_000 --npartitions 1000 --num-nodes 31 --ray --data-dir /tmp/ray --s3-bucket core-nightly-test
+
+- name: dataset_shuffle_data_loader
+  cluster:
+    app_config: shuffle_data_loader/shuffle_data_loader_app_config.yaml
+    compute_template: shuffle_data_loader/shuffle_data_loader_compute.yaml
+
+  run:
+    timeout: 7200 
```

Comment:
How long does it take now?

Reply:
huh it runs much slower with gpus as we have less gpus available...

Reply:
Hmm I see. Maybe we can make it run for an hour instead and fail it like my script?

Reply:
nvm, reduced the number of workers and repeats:
https://beta.anyscale.com/projects/prj_SVFGM5yBqK6DHCfLtRMryXHM/sessions/ses_cGNFUqzRqZAr98ZMWLb5i2g8